### PR TITLE
Updating instructions for installing KVM2 on Ubuntu & CentOS

### PR DESCRIPTION
**Pull request by abhi-g**
The current instructions on the website are already out-of-date as we noticed recently. Updating to reflect current valid procedure.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/198)

### CODE REVIEW

File: content/docs/started/getting-started-minikube.md [hunk 1]
```diff
@@ -21,22 +21,52 @@ Install [Virtual Box](https://www.virtualbox.org/wiki/Downloads) or [VMware Fusi
...
+
+# Add yourself to the libvirt group so you don't need to sudo
+# NOTE: For older Debian/Ubuntu versions change the group to `libvirtd`
+$ sudo usermod -a -G libvirt $(whoami)
+
+# Update your current session for the group change to take effect
+# NOTE: For older Debian/Ubuntu versions change the group to `libvirtd`
+$ newgrp libvirt
+```
+Then install the driver itself:
```
Comment (richardsliu) [inline content/docs/started/getting-started-minikube.md]: I remember there was a reboot step. Do we need to mention it here?

Comment (abhi-g) [reply content/docs/started/getting-started-minikube.md]: I believe the reboot was needed only to ensure that user's new group membership takes effect. That is actually achieved here by the 'newgrp libvirt' command.

**Pull request merged**